### PR TITLE
show kube-aws-ingress-controller and skipper in action

### PR DESCRIPTION
**Pull request by szuecs**
*Description of changes:*

This adds a new section  [kube-aws-ingress-controller](https://github.com/zalando-incubator/kube-ingress-aws-controller) and [skipper](https://github.com/zalando/skipper) in the ingress-controllers section.

It shows how to integrate with AWS services:
* ALB
* ACM

It shows features applied from ingress and implemented by skipper:
* basic ingress
* automatic redirect from http to https
* feature toggle
* ratelimits
* green-blue deployments

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ingress-controllers/readme.adoc [hunk 1]
```diff
@@ -210,4 +210,325 @@ of the Ingress resource:
...
+
+Cloud Labels are required to make Kube AWS Ingress Controller work,
+because it has to find the AWS Application Load Balancers it manages
+by AWS Tags, which are called cloud Labels in Kops.
+
+First you set some environment variables to choose AZs to deploy to,
+your S3 Bucket name for Kops configuration and you Kops cluster name:
+
+
+        export AWS_AVAILABILITY_ZONES=eu-central-1b,eu-central-1c
```
Comment (arun-gupta) [inline ingress-controllers/readme.adoc]: `AWS_AVAILABILITY_ZONES` should already be setup as mentioned in pre-reqs. Can you state that accordingly?

File: ingress-controllers/readme.adoc [hunk 2]
```diff
@@ -210,4 +210,325 @@ of the Ingress resource:
...
+Check in the EC2 console to ensure your ALB has been deleted.
+
+== Kube AWS Ingress Controller and Skipper
+
+link:https://github.com/zalando-incubator/kubernetes-on-aws[Kube AWS Ingress Controller]
+creates AWS Application Load Balancer (ALB) that is used to terminate TLS connections and use
+link:https://aws.amazon.com/certificate-manager/[AWS Certificate Manager (ACM)] or
+link:http://docs.aws.amazon.com/IAM/latest/APIReference/Welcome.html[AWS Identity and Access Management (IAM)]
+certificates. ALBs are used to route traffic to an Ingress http router for example
+link:https://github.com/zalando/skipper/[skipper], which routes
```
Comment (arun-gupta) [inline ingress-controllers/readme.adoc]: Highlight key value of skipper?

Comment (szuecs) [reply ingress-controllers/readme.adoc]: It's written "green-blue deployments, feature toggles, reate limits, circuitbreakers, shadow traffic or A/B tests."

File: ingress-controllers/readme.adoc (hunk 1 above)
Comment (szuecs) [reply ingress-controllers/readme.adoc]: I hope this is now more clear

**Pull request merged**